docs/deep-dives/mobile/nimbus-cli/100-configuration-variables.mdx

```diff
@@ -16,9 +16,20 @@ export ADB_PATH="/Applications/Android Studio.app/platform-tools/bin/adb"
 export XCRUN_PATH="$HOME/Library/CustomXcode/bin/xcrun"
 ```
 
-## Remote Settings
+## Experimenter URLs
 
-Remote Settings is hardcoded into `nimbus-cli` for both the stage and release versions. However, just occassionally you may want to change the URLs.
+Experimenter URLs are used for accessing experiments via the `/api/v6` endpoints. They're hardcoded into `nimbus-cli` for both stage and production versions. However, just occassionally you may want to change these URLs.
```

Comment:
Also worth noting that Draft experiments can't be guaranteed to be in a usable state so it's generally better to use Preview for testing since it passes the validation step that ensures it's in a state a client can consume.